### ISSUE TITLE
forcibly logged out, cannot log back in

### ISSUE THREAD

**Issue by staticanimal** [dimdenGD/OldTweetDeck#250]
extension was working since the most recent update but seems to have broken throughout the course of today. attempting to load tweetdeck brings up tweetdeck's loading screen with a missing loading circle and then redirects to tweetdeck's login page with an error stating "Please check your network connection and refresh TweetDeck."

clicking the log in button redirects to a twitter error page that says "Something went wrong, but don’t fret — let’s give it another shot." and a "try again" button that does nothing when clicked.

OS: Windows 11
Browser: Microsoft Edge 124.0.2478.109
Extension Version: 3.6.2

**Comment by archeruwu** [comment]
use x.com/i/tweetdeck instead of the twitter url

**Comment by staticanimal** [comment]
> use x.com/i/tweetdeck instead of the twitter url

that didn't do the trick sadly. same exact issue.

**Comment by PowerOfRed17** [comment]
I was having all the same issues on Firefox Nightly even though 3.6.2 worked perfectly yesterday. At first it was a browser issue that had to do with the new domain, but the Firefox devs fixed that issue and it was still not working for me. What ended up fixing it was disabling the Disconnect add-on (which is a redundant add-on anyway).

Check and see if you have an add-on doing something similar to Disconnect that might be causing the issue. Once disabled, I cleared cache, logged in again, refreshed everything and it all worked perfectly using the x.com url (though I did need to re-import my OTD settings). Hopefully that works for you.
